doc/quick_start.rst

```diff
@@ -0,0 +1,92 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _quick_start:
+
+
+##########################################
+How to start with modeling
+##########################################
+
+The installation procedure installs PyPSA-Earth model with all the software dependencies needed to build and run it. To properly model any region of the Earth, PyPSA-Earth needs to download and fetch different datasets. This section explains how to perform this data management.
+
+Build the tutorial model
+------------------------------
+
+A tutorial data kit was developed to facilitate exploring the model. You can use it by using the tutorial configuration file `config.tutorial.yaml` (placed in the project folder `pypsa-earth`). To do that, you may want to do a reserve copy of your current configuration file and then overwrite it by a tutorial configuration:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % cp config.tutorial.yaml config.yaml
+
+
+In the configuration file `config.yaml` there is a flag `retrieve_databundle` which triggers data loading and a `tutorial` flag which determines that the loaded data belong to the light tutorial kit:
+
+.. code:: yaml
+
+    tutorial: true
+    ...
+    retrieve_databundle: true
+
+It's recommended to set `retrieve_databundle: true` when building the model first time to download all the common data files needed. When the first run is completed and all the necessary data extracted, it may be a good idea to set `retrieve_databundle: false` to avoid data loss.
+
+How to run the model?
+------------------------------
+
+After configuration set-up, you just need run the modeling workflow with the following command:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake -j 1 solve_all_networks
+
+.. TODO Explain settings of the tutorial case
+
+This command will trigger loading of the whole dataset needed to build the model for a tutorial case if both `tutorial` and `retrieve_databundle` flags are on. The tutorial model will run simulation of power systems in Nigeria and Benin. Note please that data load will need about 1.6Gb and model building will take a while (about 20..50 minutes).
+
+Before to actually run the workflow you may check how it will look by using `dryrun` Snakemake option:
+
+.. code:: bash
+
+    .../pypsa-earth (pypsa-earth) % snakemake -j 1 solve_all_networks --dryrun
```

Comment:
Change the order. First the dry-run & then the actual execution. (requires some text adjustments)